test/parallel/test-tls-env-extra-ca-no-crypto.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const common = require('../common');
+const fixtures = require('../common/fixtures');
+const assert = require('assert');
+const { fork } = require('child_process');
+
+// This test ensures that trying to load extra certs won't throw when
```

Comment:
Since there is no test IMHO the description should be:
```
... won't throw even when there is no... Hence we don't test `common.hasCrypto`
                ^^^^
```

Reply:
Thanks. I guess I made this test too confusing.